### PR TITLE
Fix browser store interference with redux-offline

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: store/index.js [hunk 1]
```diff
@@ -91,11 +93,11 @@ export default function configureStore(initialState, persistorStorage = null) {
                     observable.subscribe({
                         next: (args) => {
                             if(args.key && args.key.indexOf(KEY_PREFIX) === 0){
-                              var keyspace = args.key.substr(KEY_PREFIX.length)
+                                var keyspace = args.key.substr(KEY_PREFIX.length);
```
Comment (enahum) [inline store/index.js]: nothing major but can we change this to const?

File: actions/storage.js [hunk 2]
```diff
@@ -79,3 +79,21 @@ export function actionOnItemsWithPrefix(prefix, action) {
...
+                storage = JSON.parse(incoming.storage)
+            } catch (err) {
+                if (process.env.NODE_ENV !== 'production') console.warn(`Error rehydrating data for key "storage"`, err)
+            }
+            dispatch({
+                type: StorageTypes.STORAGE_REHYDRATE,
+                data: storage
+            });
+        }
+        return {data: false};
```
Comment (enahum) [inline actions/storage.js]: why is this returning false?

File: store/index.js (hunk 1 above)
Comment (jespino) [reply store/index.js]: Sure!

File: actions/storage.js (hunk 2 above)
Comment (jespino) [reply actions/storage.js]: Sorry, was based on the previous code, and the previous code was an error on my understanding of the async actions expected returns. I fixed this and the previous actions.

**Pull request merged**